sass/icons/_icons.scss

```diff
@@ -18,6 +18,10 @@
 
 .icon-arrow-up-24 { background-image: url('data:image/svg+xml;charset=US-ASCII,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22utf-8%22%3F%3E%3C!DOCTYPE%20svg%20PUBLIC%20%22-%2F%2FW3C%2F%2FDTD%20SVG%201.1%2F%2FEN%22%20%22http%3A%2F%2Fwww.w3.org%2FGraphics%2FSVG%2F1.1%2FDTD%2Fsvg11.dtd%22%3E%3Csvg%20version%3D%221.1%22%20id%3D%22Layer_1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20x%3D%220px%22%20y%3D%220px%22%20%20width%3D%2224px%22%20height%3D%2224px%22%20viewBox%3D%220%200%2024%2024%22%20enable-background%3D%22new%200%200%2024%2024%22%20xml%3Aspace%3D%22preserve%22%3E%3Cg%3E%20%3Cpolygon%20fill%3D%22%23333333%22%20points%3D%2217.6%2C15.4%2012%2C9.7%206.4%2C15.4%205.6%2C14.6%2012%2C8.3%2018.4%2C14.6%20%20%22%2F%3E%3C%2Fg%3E%3C%2Fsvg%3E'); background-repeat: no-repeat; }
 
+.icon-arrow-right-circle-24 {
+  background-image: url('data:image/svg+xml;charset=US-ASCII,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22utf-8%22%3F%3E%0A%3C%21--%20Generator%3A%20Adobe%20Illustrator%2017.1.0%2C%20SVG%20Export%20Plug-In%20.%20SVG%20Version%3A%206.00%20Build%200%29%20%20--%3E%0A%3C%21DOCTYPE%20svg%20PUBLIC%20%22-//W3C//DTD%20SVG%201.1//EN%22%20%22http%3A//www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd%22%3E%0A%3Csvg%20version%3D%221.1%22%20id%3D%22Layer_1%22%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20xmlns%3Axlink%3D%22http%3A//www.w3.org/1999/xlink%22%20x%3D%220px%22%20y%3D%220px%22%0A%09%20width%3D%2224px%22%20height%3D%2224px%22%20viewBox%3D%220%200%2024%2024%22%20enable-background%3D%22new%200%200%2024%2024%22%20xml%3Aspace%3D%22preserve%22%3E%0A%3Cg%3E%0A%09%3Cpath%20fill%3D%22%23333333%22%20d%3D%22M12%2C3c2.3%2C0%2C4.6%2C0.9%2C6.4%2C2.6c3.5%2C3.5%2C3.5%2C9.2%2C0%2C12.7C16.6%2C20.1%2C14.3%2C21%2C12%2C21s-4.6-0.9-6.4-2.6%0A%09%09c-3.5-3.5-3.5-9.2%2C0-12.7C7.4%2C3.9%2C9.7%2C3%2C12%2C3%20M12%2C2C9.3%2C2%2C6.8%2C3%2C4.9%2C4.9C1%2C8.8%2C1%2C15.2%2C4.9%2C19.1C6.8%2C21%2C9.3%2C22%2C12%2C22%0A%09%09c2.7%2C0%2C5.2-1%2C7.1-2.9c3.9-3.9%2C3.9-10.2%2C0-14.1C17.2%2C3%2C14.7%2C2%2C12%2C2L12%2C2z%22/%3E%0A%09%3Cpolyline%20fill%3D%22none%22%20stroke%3D%22%23333333%22%20stroke-miterlimit%3D%2210%22%20points%3D%2210.8%2C7.5%2015.2%2C12%2010.8%2C16.5%20%09%22/%3E%0A%3C/g%3E%0A%3C/svg%3E');
```

Comment:
Maintaining consistency in formatting would be nice. Like on L19, it looks like it's all in one line.

Same comment applies to the addition made in file dist/sass/icons/_icons.scss